tests/fixups/TraceFixup/DynamicLinkLibraryFixup.cpp

```diff
@@ -13,13 +13,39 @@
 auto AddDllDirectoryImpl = &::AddDllDirectory;
 DLL_DIRECTORY_COOKIE __stdcall AddDllDirectoryFixup(_In_ PCWSTR newDirectory)
 {
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);
     auto entry = LogFunctionEntry();
     auto result = AddDllDirectoryImpl(newDirectory);
+	QueryPerformanceCounter(&TickEnd);

```

Comment:
this pattern could be factored into an RAII object that 'wraps' the impl call

Reply:
Not sure the ask here.  Must be above my pay grade.  Can you supply an example of what you are looking for here?